dev/dev.ex

```diff
@@ -46,4 +46,340 @@ defmodule Dev do
       end
     end
   end
+
+  @ig_contacts [
```

Comment:
you'd need to regenerate this list and edit it manually since when you run it, it'd likely be different